code/game/jobs/job/support_chaplain.dm

```diff
@@ -35,7 +35,7 @@
 	var/obj/item/weapon/storage/bible/B = new /obj/item/weapon/storage/bible(H)
 
 	spawn()
-		H.equip_to_slot_or_del(B, slot_l_hand)
+		H.equip_to_slot_or_del(B, slot_r_hand)
```

Comment:
Maybe 
```
if(!H.put_in_hands(B))
     qdel(B)
```
?

Reply:
https://github.com/ParadiseSS13/Paradise/blob/d2eeac3cf8a32b9ed790e7785b529d3958453ef9/code/modules/mob/mob.dm#L230 Honestly you should use the `equip_or_collect` proc instead of this one, because what if someone later on for some reason makes the vox tank spawn in their right hand?

Reply:
@KasparoVy @Citinited  The problem with that is that further down we update a specific hand. So if we don't know where we put it, we'll need to find it before we update. It would probably be easier to modify the voxygen tank spawn code.

Reply:
the proc I posted apparently drops the bible on the floor, so you shouldn't need to worry about having a free hand.

Reply:
@Citinited The problem is that in line 151 it updates the bible to the custom version decided by the GUI:
`H.update_inv_r_hand() // so that it updates the bible's item_state in his hand`
That code needs to know the location of the bible to update it. If we just put it in any available spot we'll need to add some code to find where the bible is before we update it.

Reply:
But the `item_state` of the bible is already being changed in the big `while` loop, and when they take the bible out of their bag then their mob sprite will automatically be updated.

Reply:
Additionally, why can't we just move this line down below the logic that determines what the bible's final icons are? :laugh:

Reply:
It's worth noting that the loc of an object doesn't matter because you can call a global update inv on a mob to catch everything, or alternatively, you could just do something like this:
```if(B.loc == H.contents) //covers both hands
    H.update_inv()